tests/storage/cdi_import/conftest.py

```
@@ -136,37 +132,31 @@ def running_pod_with_dv_pvc(
 
 
 @pytest.fixture()
```

Ruff (0.14.14)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)